apps/ssr-test-app/package.json

```diff
@@ -8,7 +8,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "^15.1.6",
+    "next": "^15.2.3",
```

Comment:
the next package seems to constantly trigger a CG upgrade. Could you double check that all of the instances in the pnpm-lock file reference this new version?

Reply:
Thanks for calling this out, I looked and decided to upgrade the eslint-plugin-next package as well, even though it wasn't pulling in any vulnerable version. Looks like all the lockfile references are updated to 15.2.4